### PR TITLE
for xma backward compatibility

### PR DESCRIPTION
**Pull request by sarab96**
for xma backward compatibility

### CODE REVIEW

File: src/xma/include/app/xmahw.h [hunk 1]
```diff
@@ -43,6 +45,7 @@ typedef enum {
   XMA_CMD_STATE_PSK_CRASHED = 7, //PS kernel has crashed
   XMA_CMD_STATE_MAX = 8 // Always the last one
 } XmaCmdState;
+static_assert(sizeof(XmaCmdState) <= 4, "XmaCmdState size must be less than 5 bytes");
```
Comment (sonals) [inline src/xma/include/app/xmahw.h:48]: `static_assert(sizeof(XmaCmdState) <= sizeof(int32_t), "XmaCmdState size must be less than 4 bytes");`

**Review by sonals — CHANGES_REQUESTED** [review]
Just update the assert

File: src/xma/include/app/xmahw.h (hunk 1 above)
Comment (stsoe) [reply src/xma/include/app/xmahw.h:48]: Move the static_assert to a .cpp file.

Comment (sarab96) [reply src/xma/include/app/xmahw.h]: Done

**Review by sonals — APPROVED** [review]

**Review by stsoe — APPROVED** [review]

**Pull request merged**